.github/workflows/performance.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
             - name: Compare performance with trunk
```

Comment:
Maybe we should change the name of the step
```suggestion
            - name: Compare performance with base branch
```

Reply:
Yep, done.